test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -78,4 +79,31 @@ describe('POST /chat', () => {
     expect(message.message.id).to.exist;
     expect(memberWithNotification.newMessages[`${group._id}`]).to.exist;
   });
+
+  it('Returns an error when the user has been posting too many messages', async () => {
+    let spamError = {};
+
+    // this loop should only actually run 10 times if the test would fail, else it is cut short at first error
+    for (let i = 0; i < 5; i++) {
+      try {
```

Comment:
okay, I had only done this to make it so the test wouldn't have to be changed every time we change the threshold but an explicit amount of calls would be a better test I will change that.